index.html

```diff
@@ -260,13 +260,13 @@ <h4>Anual</h4>
                     </div>
                 </div>
 
-                <div class="row optional-add-on-row">
-                    <div class="col-md-12">
-                        <div>
-                            <p>*Opcional: Agregar una laptop por RD$10,600</p>
-                        </div>
-                    </div>
-                </div>
+<!--                <div class="row optional-add-on-row">-->
```

Comment:
Would you mind rather removing this block? I see no use in leaving it as a comment. Thanks!